### PR TITLE
Add links to Q&A

### PR DESCRIPTION
**Pull request by BillWagner**
Each landing page where there was a relevant tag is linked to the relevant locations on Q&A. Exceptions are:

- dotnet-azure doesn't have a tag.
- installers have different tags for different installers, so multiple links have been added.

### CODE REVIEW

**Comment by jamesmontemagno** [comment]
Lovely! Anything for Xamarin?I think for Azure, we don't have like .NET SDK for Azure (we could create one), however all azure products are at: https://docs.microsoft.com/answers/products/azure?product=all

**Comment by BillWagner** [comment]
@jamesmontemagno 

> Anything for Xamarin?I

That's in a different repo. I think @davidbritch would add those links from the Xamarin docs repos.

**Comment by jamesmontemagno** [comment]
I know one other thing is on the bottom of atleast Xamarin docs we have ->
image

I would love to to see:
* Stack Overflow change to Microsoft Q&A
* Developer Community... I think should point to -> https://dotnet.microsoft.com/platform/community

**Comment by BillWagner** [comment]
@jamesmontemagno I'd make those suggestions [here](https://github.com/MicrosoftDocs/xamarin-docs/issues)

**Comment by jamesmontemagno** [comment]
great point! I can get them in :)

**Review by jamesmontemagno — APPROVED** [review]

**Review by luisquintanilla — COMMENTED** [review]
LGTM

File: docs/core/install/index.yml [hunk 1]
```diff
@@ -66,3 +66,19 @@ landingContent:
             url: linux-sles.md
           - text: Ubuntu
             url: linux-ubuntu.md
+
+# Card
+  - title: "Q&A"
+    linkLists:
+      - linkListType: get-started
+        links:
+          - text: standalone installers
```
Comment (adegeo) [inline docs/core/install/index.yml:75]:
```suggestion
          - text: Standalone installers
```

**Review by adegeo — APPROVED** [review]

**Pull request merged**